tests/transforms/test_metric_tensor.py

```diff
@@ -703,7 +703,7 @@ def test_no_trainable_params_tape(self):
         assert res == ()
 
 
-fixed_pars = np.array([-0.2, 0.2, 0.5, 0.3, 0.7], requires_grad=False)
+fixed_pars = [-0.2, 0.2, 0.5, 0.3, 0.7]

```

Comment:
Is it raising a warning?

Reply:
Yes, because the `fubini_ansatz#` ansatze are used with the interfaces, and using Autograd parameters mixed with TF/JAX/Torch parameters raises a warning in the `math/utils.py` file when using multi-dispatch functions.